src/lib/scripts/verseSelectUtil.ts

```diff
@@ -0,0 +1,105 @@
+import { selectedVerses } from '$lib/data/stores';
+import config from '$lib/data/config';
+
+export type SelectedReference = {
+    docSet: string;
+    book: string;
+    chapter: string;
+    verse: string;
+};
+export function onClickText(e) {
+    let target = e.target;
+
+    while (!isSelectableText(target) && !isClickable(target) && !isMain(target)) {
+        target = target.parentNode;
+    }
+    if (isSelectableText(target)) {
+        const id = removeIdSuffixes(target.id);
+        if (target.className.indexOf('selected') < 0) {
```

Comment:
This can be:
`if (!target.classList.contains('selected'))`

Reply:
Done